src/analytics/Prices.js

```diff
@@ -0,0 +1,82 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row } from 'reactstrap';
+
+import ErrorBoundary from 'components/ErrorBoundary';
+import { compareStrings, SortableTable } from 'components/SortableTable';
+import { cardEtchedPrice, cardFoilPrice, cardNormalPrice, cardPrice, cardStatus } from 'utils/Card';
+
+const Prices = ({ cards }) => {
+  const getValue = (collection, allowedStatus, allowedFinish) => {
+    const allowedStatuses = [];
+    if (allowedStatus === 'Owned' || allowedStatus === 'Any') allowedStatuses.push('Ordered', 'Owned', 'Premium Owned');
+    if (allowedStatus === 'Not Owned' || allowedStatus === 'Any') allowedStatuses.push('Not Owned', 'Proxied');
+
+    let priceFn;
+    switch (allowedFinish) {
+      case 'Non-foil':
+        priceFn = cardNormalPrice;
+        break;
+      case 'Foil':
+        priceFn = cardFoilPrice;
+        break;
+      case 'Etched':
+        priceFn = cardEtchedPrice;
+        break;
+      default:
+        priceFn = cardPrice;
+    }
+
+    return '$'.concat(
+      parseFloat(
+        collection
+          .map((card) => {
+            if (allowedStatuses.find((status) => status === cardStatus(card))) return priceFn(card) ?? 0;
```

Comment:
The condition can be simplified as a call to `allowedStatuses.includes`, and possibly removed in favor of a `.filter` call prior to the map, depending on personal preference.

The `map` call also isn't strictly necessary, you can get the price item in the reducer, but I suppose that's also more of a question of taste.